### PR TITLE
v0.3.3-alpha.0

### PR DESCRIPTION
**Pull request by koonopek**
* Testing if publishing is working correctly (with alpha version) - works
* Had to add in every package.json publishConfig: access: public, because packages starting from `@org-name` are by default private

### CODE REVIEW

File: packages/starknet-connector/package.json [hunk 1]
```diff
@@ -12,13 +12,16 @@
...
   },
-  "devDependencies": {},
   "files": [
     "dist/**/*"
-  ]
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "gitHead": "c7b70dfbd7450063c8aabb891b2fc46e75df440a"
```
Comment (rpiszczatowski) [inline packages/starknet-connector/package.json:26]: Should we really have 'gitHead' property in our repo.
I haven't found lots of documentation on it, but according to this: https://github.com/lerna/lerna/issues/1880 'gitHead' should be a transient property only set for publishing time.

File: packages/starknet-connector/package.json [hunk 2]
```diff
@@ -12,13 +12,16 @@
...
     "ethers": "^5.7.2",
     "starknet": "^5.19.5"
   },
-  "devDependencies": {},
   "files": [
     "dist/**/*"
-  ]
+  ],
+  "publishConfig": {
+    "access": "public"
```
Comment (rpiszczatowski) [inline packages/starknet-connector/package.json:24]: Do we really want to publish starknet-connector, starknet-relayer and ton-connector?

File: packages/fuel-connector/package.json [hunk 3]
```diff
@@ -24,5 +24,9 @@
   "files": [
     "dist/src/**/*",
     "src/autogenerated/prices.bin"
-  ]
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "gitHead": "c7b70dfbd7450063c8aabb891b2fc46e75df440a"
```
Comment (rpiszczatowski) [inline packages/fuel-connector/package.json:31]: Do we want to publish this connector?

File: packages/eth-contracts/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redstone-finance/eth-contracts",
-  "version": "0.3.0",
+  "version": "0.3.2-alpha.0",
```
Comment (rpiszczatowski) [inline packages/eth-contracts/package.json:3]: Really 0.3.2? Other packages have 0.3.3.

**Review by rpiszczatowski — CHANGES_REQUESTED** [review]
Couple of questions.

File: packages/starknet-connector/package.json (hunk 2 above)
Comment (koonopek) [reply packages/starknet-connector/package.json:24]: Why not? @Lukasz2891

File: packages/fuel-connector/package.json (hunk 3 above)
Comment (koonopek) [reply packages/fuel-connector/package.json:31]: Why not? @Lukasz2891

File: packages/starknet-connector/package.json (hunk 1 above)
Comment (koonopek) [reply packages/starknet-connector/package.json:26]: So it wasn't cleared by lerna :(

File: packages/starknet-connector/package.json (hunk 2 above)
Comment (koonopek) [reply packages/starknet-connector/package.json:24]: Also it is already published. I haven't configured what should be published in this PR it was done on first PR on lerna

File: packages/eth-contracts/package.json (hunk 4 above)
Comment (koonopek) [reply packages/eth-contracts/package.json]: fixed! Good catch

File: packages/starknet-connector/package.json (hunk 1 above)
Comment (koonopek) [reply packages/starknet-connector/package.json]: Removed

**Review by rpiszczatowski — APPROVED** [review]

**Pull request merged**